src/main/resources/templates/targets/aws-s3-target-template.yaml

```diff
@@ -23,6 +26,22 @@ target:
   lifecycleHooks:
     delete:
       - hookName: deleteLocalRepoFolderLifecycleHook
+      - hookName: deleteIndexLifecycleHook
+    {{#if source_target}}
+    duplicate:
+      - hookName: duplicateIndexLifecycleHook
+      - hookName: duplicateProcessCommitsFileLifecycleHook
+      - hookName: duplicateS3LifecycleHook
+        region: ${aws.region}
+        accessKey: ${aws.accessKey}
+        secretKey: ${aws.secretKey}
+        url: {{aws.s3.url}}
+        endpoint: {{aws.s3.endpoint}}
+        pathStyleAccess: {{aws.s3.path_style_access}}
+        sourceConfig:
+          url: {{source.aws.s3.url}}
+          localRepoPath: {{source.local_repo_path}}
```

Comment:
Can't remember if we agreed to add blob store duplication as part of the duplicate hooks. If we agreed to not do it, this is fine.